sdk/typescript/src/keypairs/passkey/keypair.ts

```diff
@@ -81,13 +84,24 @@ export class BrowserPasskeyProvider implements PasskeyProvider {
         })) as RegistrationCredential;
     }
 
-    async get(challenge: Uint8Array): Promise<AuthenticationCredential> {
+    async get(
+        challenge: Uint8Array,
+        allowCredentialIds?: Uint8Array[],
+    ): Promise<AuthenticationCredential> {
         return (await navigator.credentials.get({
             publicKey: {
                 challenge,
                 userVerification:
                     this.#options.authenticatorSelection?.userVerification || 'required',
                 timeout: this.#options.timeout ?? 60000,
+                ...(allowCredentialIds && allowCredentialIds.length
+                    ? {
+                          allowCredentials: allowCredentialIds?.map((id) => ({
+                              type: 'public-key' as const,
+                              id: id,
+                          })),
+                      }
```

Comment:
Just some code simplification, not a big deal
```suggestion
                ...(allowCredentialIds?.length
                    ? {
                          allowCredentials: allowCredentialIds.map((id) => ({
                              type: 'public-key',
                              id,
                          })),
                      }
```